perf/benchmark/flame/README.md

```diff
@@ -66,3 +66,27 @@ Flame graphs are created from data collected using linux `perf_events` by the `p
     generating svg file svc05-0-7-564865d756-pvjhn-2020-01-29-22-34-19.perf
     ...
     ```
+
+## Run pprof tool (istiod)
+
+Flame graphs are created from data collected by pprof profiles
+
+1. (Optional) Find the istiod pod you want to profile using kubectl and specify the pod and namesapce in the following command usting the `-p` and `-n` flags.
+   By default, the first pod in istio-system with label `app=istiod` will be used.
+
+1. Run [`get_istiod_pprof.sh`](get_istiod_pprof.sh) to generate the flame graph svg. The svg file should be created under `flameoutput` dir
+
+    ```plain
+    ./get_istiod_pprof.sh
```

Comment:
change `./get_istiod_pprof.sh` to `./get_istiod_pprof.sh -p istiod-67bff5dbbf-grl94 -n istio-system -d 20`?